packages/fiori-elements-writer/src/types.ts

```diff
@@ -29,7 +29,37 @@ export class ValidationError extends Error {
 }
 
 export interface EntityConfig {
-    mainEntityName: string; // Defines the main list page entity
+    /**
+     * The name of the main entity used for list page entity
+     */
+    mainEntityName: string;
+    /**
+     * Represents a parameter used alongside the main entity to query for data.
+     * When this parameter is set, the prompt for selecting navigation properties is skipped,
+     * as the entity set is parameterised and directly linked to its target entity set.
+     * If a value is provided, navigation routes will always include it.
+     *
+     * @example
```

Comment:
Might be worth showing the inputs here also for clarity?
```suggestion
     * @example
     {
         mainEntityName: `Products`,
         mainEntityParameterName: 'ProductFIlter'
     }
```